<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterPersonel</class>
 <widget class="QDialog" name="AjouterPersonel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/sig/erp.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="iconnes.qrc">:/sig/images/erp.jpg</pixmap>
   </property>
  </widget>
  <widget class="QDateEdit" name="entreeDE">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="foctionCOB">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>480</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Directeur</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Secrétaire</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chef Maintenance</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Assistant chef maintenance</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Directeur Hébergement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chef Réception</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Réceptionniste</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chef cuisinier</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Assistant du chef cuisinier</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Autre</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>295</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deconnectionPB">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>180</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Déconnexion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>-40</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Aymen/Downloads/logo Abou Nawes.gif</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Entrée:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="motifCOB">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fin CDD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Démission</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Retraite</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Licenciement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Décès</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Autre</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="naissanceDE">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>335</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>630</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="motifL">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Motif:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="arretDE">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>560</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>445</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>N° tel:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="arretL">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrêt:  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>480</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Fonction: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>333</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Naissance:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimerPB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CIN:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enregistrerPB">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>385</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sexe:</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>20</width>
     <height>561</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>0</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Desktop/outils/Sans titre.jpg</pixmap>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>1351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="identifiantLE">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenomLE">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomLE">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="adresseLE">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numLE">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLE">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="MaleRB">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>340</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="femaleRB">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>340</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QTableView" name="employesTV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>351</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RechercherGrhPB">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher/ Modifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RechercherGrhLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="actualiserPB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="SortieCL">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer la sortie de l'Employe:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enregistrerPB2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>610</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iconnes.qrc"/>
 </resources>
 <connections/>
</ui>
